src/groovy/org/pih/warehouse/invoice/InvoiceStatus.groovy

```diff
@@ -11,15 +11,19 @@ package org.pih.warehouse.invoice
 
 enum InvoiceStatus {
 
-    PENDING(0),  // Drafted
-    INVOICED(10), // Invoice has been issued / received
-    SUBMITTED(20), // Invoice has been submitted for approval
-    POSTED(30),  // Invoice has been posted
-    PAID(40) // Invoice has been paid
+    PENDING(0, 'warning'),  // Drafted
```

Comment:
I think I don't like this. Could you add this as a StatusType enum at least? (and remember to use it on the OrderSummaryStatus too). And please add a comment in it to describe what it is for.

```
enum StatusType {
  PRIMARY,
  SUCCESS,
  WARNING,
  DANGER
}
```